utm.yaml

```diff
@@ -988,7 +994,6 @@ components:
         Details of a UTM Operation. Note that this data is not stored in the DSS; only with the clients.
```

Comment:
State is stored by the DSS. Reword to: "Note that the exact volumes are not stored in the DSS; only with the clients."

Reply:
I think the original text is still accurate as State is no longer a member of OperationDetails (it has been moved to OperationReference).

Reply:
Makes sense. No change needed. I'll resolve this.

Reply:
@BenjaminPelletier please resolve (I can't since I don't have write access).